package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "package.json",
+  "version": "1.0.0",
```

Comment:
Please have a look at this and remember to version your projects! :)
https://semver.org/